GSA/COM API/.NET/GsaComHelper/DotNetHelpers.vbproj

```
@@ -34,21 +34,11 @@
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>true</BootstrapperEnabled>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x64' ">
-    <!-- paths to libraries -->
-    <InteropDllPath>..\..\programs64_d\Interop.Gsa_10_0.dll</InteropDllPath>
-    <InteropDllPathCopy>..\GsaCOM\Debug\x64\</InteropDllPathCopy>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x64' ">
-    <!-- paths to libraries -->
-    <InteropDllPath>..\programs64\Interop.Gsa_10_0.dll</InteropDllPath>
-    <InteropDllPathCopy>..\gsrevit\GsaCOM\Release\x64\</InteropDllPathCopy>
-  </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
     <DebugSymbols>true</DebugSymbols>
     <DefineDebug>true</DefineDebug>
     <DefineTrace>true</DefineTrace>
-    <OutputPath>.\</OutputPath>
+    <OutputPath>..\..\programs64_d\</OutputPath>

```

Not relevant externally
```suggestion
    <OutputPath>.\</OutputPath>
```